### PR TITLE
Squash on Green

### PR DESCRIPTION
**Pull request by ashfurrow**
This PR re-applies the work from #161 (reverting #163) and fixing the syntax issue that caused the revert in the first place in b366023. I pinned the TypeScript version to what's actually used by Peril to avoid this kind if unsupported-syntax-error from happening in the future.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,20 +22,20 @@
...
+    "jest": "^26.4.2",
     "jira-client": "^6.4.1",
     "lint-staged": "^7.2.2",
-    "prettier": "^1.15.2",
-    "ts-jest": "^24.0.0",
+    "prettier": "^2.1.1",
+    "ts-jest": "^26.3.0",
     "ts-node": "^7.0.1",
-    "typescript": "^3.1.6"
+    "typescript": "3.4.1"
```
Comment (jonallured) [inline package.json:38]: **question [non-blocking]:** typescript version pin

This is the pin you mentioned? Couldn't one clobber this with `yarn upgrade --latest`? Is there another, more resilient way to specify which version of a package one wants? 🤔 

*format via [Conventional Comments](https://conventionalcomments.org/)*

**Review by jonallured — APPROVED** [review]
LGTM!! 🎉

File: package.json (hunk 1 above)
Comment (ashfurrow) [reply package.json:38]: This is the pin, I'm not sure about the semantics of `yarn upgrade` – I'm basing this off Renovate's PRs to pin dependencies in Eigen, which follow this format. (Previously, in c2cdda6, this was at 4.0.2.)

**Comment by ashfurrow** [comment]
Ugh! Still seeing a failure:

```
2020-09-11T19:03:11.993343+00:00 app[web.1]: Fri, 11 Sep 2020 19:03:11 GMT runFromSameHost Unable to evaluate the Dangerfile
2020-09-11T19:03:11.993345+00:00 app[web.1]:  TypeError: Cannot convert undefined or null to object
2020-09-11T19:03:11.993347+00:00 app[web.1]:     at Function.values (<anonymous>)
2020-09-11T19:03:11.993348+00:00 app[web.1]:     at Object.exports.rfc10 (peril-downloaded-artsy/peril-settings@org/mergeOnGreen.ts:26:35)
2020-09-11T19:03:11.993349+00:00 app[web.1]:     at process._tickCallback (internal/process/next_tick.js:68:7)
```

I'll have to look into this later, sorry for the noise Jon!

**Comment by jonallured** [comment]
Oh bummer - no problem though we can totally come back to this. Monday Ash is going to crush it!! ❤️

**Comment by ashfurrow** [comment]
My guess is that the relative import between files isn't setup right (because of how Peril executes things). Other relative imports do a weird `await import` dance that we need to replicate.

**Pull request merged**